src/frequenz/sdk/microgrid/__init__.py

```diff
@@ -174,53 +174,128 @@
+while considering the priority of the actors and the bounds they set, to
+calculate the target power for the components.
 
-The algorithm used for resolving power conflicts based on actor priority can be
-found in the documentation for any of the
-[`propose_power`][frequenz.sdk.timeseries.battery_pool.BatteryPool.propose_power]
-methods.
+The final target power can be accessed using the receiver returned from the
+[`power_status`][frequenz.sdk.timeseries.battery_pool.BatteryPool.power_status]
+method available for all pools, which also streams the bounds that an actor
+should comply with, based on its priority.
 
-### Shifting the target power by an Operating Point power
+#### Adding the power proposals of individual actors
 
-There are cases where the target power needs to be shifted by an operating point.  This
-can be done by designating some actors to be able to set only the operating point power.
+When an actor A calls the `propose_power` method with a power, the proposed
+power of the lower priority actor will get added to actor A's power.  This works
+as follows:
 
-When creating a `*Pool` instance using the above-mentioned constructors, an optional
-`set_operating_point` parameter can be passed to specify that this actor is special, and
-the target power of the regular actors will be shifted by the target power of all actors
-with `set_operating_point` together.
+ - the lower priority actor would see bounds shifted by the power proposed by
+   actor A.
+ - After lower priority actor B sets a power in its shifted bounds, it will get
+   shifted back by the power set by actor A.
 
-In a location with 2 regular actors and 1 `set_operating_point` actor, here's how things
-would play out:
+This has the effect of adding the powers set by actors A and B.
 
-1. When only regular actors have made proposals, the power bounds available from the
-   batteries are available to them exactly.
+*Example 1*: Battery bounds available for use: -100kW to 100kW
 
-   | actor priority | in op group? | proposed power/bounds | available bounds |
-   |----------------|--------------|-----------------------|------------------|
-   | 3              | No           | 1000, -4000..2500     | -3000..3000      |
-   | 2              | No           | 2500                  | -3000..2500      |
-   | 1              | Yes          | None                  | -3000..3000      |
+| Actor | Priority | System Bounds   | Requested Bounds | Requested    | Aggregate |
+|       |          |                 |                  | Power        | Power     |
+|-------|----------|-----------------|------------------|--------------|-----------|
+| A     | 3        | -100kW .. 100kW | None             | 20kW         | 20kW      |
+| B     | 2        | -120kW .. 80kW  | None             | 50kW         | 70kW      |
+| C     | 1        | -170kW .. 30kW  | None             | 50kW         | 100kW     |
+|       |          |                 |                  | target power | 100kW     |
```

Comment:
Maybe like [in the test](https://github.com/frequenz-floss/frequenz-sdk-python/pull/1146/files#diff-4312e57dc65aad6abe1ffd3ac6ff96fc64b446b1e0a5acfad6a13e5af1e98120R582-R596), adding a column with the "adjusted power" could add some extra clarity.

Reply:
done